src/System.Data.Odbc/src/System/Data/Odbc/Odbc32.cs

```diff
@@ -0,0 +1,997 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.Odbc {
+
+    using System;
+    using System.Data;
+    using System.Data.Common;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Runtime.InteropServices;
+    using System.Runtime.ConstrainedExecution;
+    using System.Text;
```

Comment:
Namespace imports should be specified at the top of the file, outside of namespace declarations and should be sorted alphabetically.